json/src/lib.rs

```diff
@@ -1140,6 +1140,39 @@ pub enum PubKeyOrAddress<'a> {
     PubKey(&'a PublicKey),
 }
 
+#[derive(Serialize, Deserialize, Clone, PartialEq, Eq, Debug)]
+#[serde(untagged)]
+pub enum ScanTxoutRequest {
+    Single(String),
+    Extended {
+        dest: String,
+        range: u64,
+    }
+}
+
+#[derive(Serialize, Deserialize, Clone, PartialEq, Eq, Debug)]
+pub struct ScanUtxoResult {
+    pub success: bool,
+    pub txouts: u64,
```

Comment:
In #132 I also had a `txouts` field which I rename to `tx_outs`. I chose this because of how `get_tx_out()` and `GetTxOutResult` are called, with `tx` and `out` as separate words. If we decide that `txouts` is preferred, #132 should be changed accordingly to be consistent.

Reply:
Ah, I also had a `bestblock` in #132, which I renamed to `best_block`. Same goes for that.